pkg/webui/console/containers/device-onboarding-form/device-provisioning-form-section/index.js

```diff
@@ -42,8 +48,21 @@ const initialValues = merge(
   registrationInitialValues,
 )
 
-const DeviceTypeFormSection = props => {
+const DeviceTypeFormSection = () => {
   const [isClaiming, setIsClaiming] = useState(undefined)
+  const {
+    values: {
+      _inputMethod,
+      version_ids: version,
+      frequency_plan_id,
+      lorawan_version,
+      lorawan_phy_version,
+    },
+  } = useFormContext()
+  const template = useSelector(selectDeviceTemplate)
```

Comment:
```suggestion
  const template = useSelector(selectDeviceTemplate)
```

Note my use of selector hooks here and in the other components. I think we should move towards using those by default instead of the `connect()`-HOC. The problem I have with `connect()` are:
- It's not following the hooks paradigm.
- It generates a new component only for passing through state data.
- I also don't really like outsourcing the `connect()` into it's own module. It can be nice for complex components but I typically find myself looking for the origin of certain props which is effectively hidden when looking at the component alone.